src/views/transactions/components/form-container/form.container.view.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+import Container from "src/views/shared/container/container.view";
+import useFormContainerStyles from "./form.container.styles";
+
+const FormContainer: React.FC = ({ children }) => {
+  const classes = useFormContainerStyles();
+
+  return (
+    <div className={classes.root}>
+      <Container disableTopGutter>
```

Comment:
Why use an extra wrapper for this container?

Reply:
We need to add it to add a margin from the top and to leave some spacing for the header. Also to add some responsive rules.